### PR TITLE
Providing opam files for those contribs which compile with Coq 8.9 (part 2)

### PR DESCRIPTION
**Pull request by herbelin**

### CODE REVIEW

File: released/packages/coq-huffman/coq-huffman.8.9.0/opam [hunk 1]
```diff
@@ -1,39 +1,37 @@
 opam-version: "2.0"
```
Comment (clarus) [inline released/packages/coq-huffman/coq-huffman.8.9.0/opam]: This package looks suspicious to me as it seems to conflict with an existing package. In particular there is a difference on the `date` tag.

**Comment by clarus** [comment]
I would be in favor of merging these pull-requests when the only errors are timeouts. Anyway it will be tested by the opam bench again.

File: released/packages/coq-huffman/coq-huffman.8.9.0/opam (hunk 1 above)
Comment (herbelin) [reply released/packages/coq-huffman/coq-huffman.8.9.0/opam]: Indeed, this is an error. I now removed huffman from the list.

**Comment by clarus** [comment]
Top!

**Pull request merged**